tests/e2e/framework/utils.go

```diff
@@ -613,3 +613,112 @@ func getTestMetricsCMD(namespace string) string {
 func GetPoolNodeRoleSelector() map[string]string {
 	return utils.GetNodeRoleSelector(TestPoolName)
 }
+
+// GetDefaultStorageClassProvisioner retrieves the provisioner from the default storage class
+func (f *Framework) GetDefaultStorageClassProvisioner() (string, error) {
+	var scList unstructured.UnstructuredList
+	scList.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "storage.k8s.io",
+		Version: "v1",
+		Kind:    "StorageClassList",
+	})
+
+	if err := f.Client.List(context.TODO(), &scList); err != nil {
+		return "", fmt.Errorf("failed to list storage classes: %w", err)
+	}
+
+	for _, sc := range scList.Items {
+		annotations := sc.GetAnnotations()
+		if annotations != nil {
+			if isDefault, ok := annotations["storageclass.kubernetes.io/is-default-class"]; ok && isDefault == "true" {
+				provisioner, found, err := unstructured.NestedString(sc.Object, "provisioner")
+				if err != nil {
+					return "", fmt.Errorf("failed to get provisioner from storage class: %w", err)
+				}
+				if !found {
+					return "", fmt.Errorf("provisioner field not found in default storage class")
+				}
+				return provisioner, nil
+			}
+		}
+	}
+
+	return "", fmt.Errorf("no default storage class found in the cluster")
+}
+
+// CreateCustomStorageClass creates a custom StorageClass object
+func (f *Framework) CreateCustomStorageClass(name, provisioner string) *unstructured.Unstructured {
+	sc := &unstructured.Unstructured{}
+	sc.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "storage.k8s.io",
+		Version: "v1",
+		Kind:    "StorageClass",
+	})
+	sc.SetName(name)
+	unstructured.SetNestedField(sc.Object, provisioner, "provisioner")
+	return sc
+}
+
+// AssertScanPVCHasStorageConfig verifies that the PVC for a scan has the expected storage configuration
+func (f *Framework) AssertScanPVCHasStorageConfig(scanName, namespace, expectedStorageClassName string, expectedAccessMode corev1.PersistentVolumeAccessMode) error {
```

Comment:
I have a really really old PR that added a separate file for assertions.

https://github.com/ComplianceAsCode/compliance-operator/pull/424

That way we could keep utility functions (e.g., `CreateCustomStorageClass`) separate from assertions (e.g., `AssertScanPVCHasStorageConfig`).

We could dust that off and start incorporating it here, since it helps keep things clearer, and the overall size of the files smaller, since all the utilities and assertions don't need to live in a single file.